db_stress_tool/db_stress_test_base.cc

```diff
@@ -862,6 +873,29 @@ Status StressTest::TestIterate(ThreadState* thread,
   return s;
 }
 
+// Test the return status of GetLiveFiles, GetSortedWalFiles, and
+// GetCurrentWalFile. Each time, randomly select one of them to run
+// and return the status.
+Status StressTest::VerifyGetLiveAndWalFiles(ThreadState* thread) {
+  int case_num = thread->rand.Next() % 3;
+  if (case_num == 0) {
+    std::vector<std::string> live_file;
+    uint64_t manifest_size;
+    return db_->GetLiveFiles(live_file, &manifest_size);
+  }
+
+  if (case_num == 1) {
+    VectorLogPtr log_ptr;
+    return db_->GetSortedWalFiles(log_ptr);
+  }
+
+  if (case_num == 2) {
+    std::unique_ptr<LogFile> cur_wal_file;
+    return db_->GetCurrentWalFile(&cur_wal_file);
+  }
+  return Status::OK();
```

Comment:
Does it happen? If not assert(false).

Reply:
@siying Sure, just in case thread->rand has something wrong.